shopfloor/services/single_pack_transfer.py

```diff
@@ -191,19 +192,25 @@ def _create_package_level(self, package):
         picking.action_assign()
         return package_level
 
-    def _is_move_state_valid(self, move):
-        return move.state != "cancel"
-
-    def _is_dest_location_valid(self, move, scanned_location):
-        """Forbid a dest location to be used"""
+    def _is_move_state_valid(self, moves):
+        return all(move.state != "cancel" for move in moves)
+
+    def _is_dest_location_valid(self, moves, scanned_location):
+        """We ensure the destination is either valid regarding the picking
```

Comment:
Better to have a summary sentence and the details separated by a blank line to be inline with PEP8:
```suggestion
        """Ensure the destination location is valid for given moves.

        We ensure the destination is either valid regarding the picking [...]
        """
```

Reply:
fixed